app/_config/injector.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: 'addonsinjector'
+---
+SilverStripe\Core\Injector\Injector:
+  AddonBuilder:
+    constructor:
+      - '%$PackagistService'
+  AddonUpdater:
+    constructor:
+      - '%$PackagistService'
+      - '%$Heyday\Elastica\ElasticaService'
+      - '%$Composer\Package\Version\VersionParser'
+  SilverStripe\Control\RequestProcessor:
+    properties:
+      filters:
+        - '%$SiteErrorPageFilter'
+  SilverStripeVersionUpdater:
+    constructor:
+      - '%$PackagistService'
+  UpdateAddonsTask:
+    constructor:
+      - '%$AddonUpdater'
+  UpdateSilverStripeVersionsTask:
+    constructor:
+      - '%$SilverStripeVersionUpdater'
+  BuildAddonsTask:
+    constructor:
+      - '%$AddonBuilder'
+
+  Psr\SimpleCache\CacheInterface.addons:
+    factory: SilverStripe\Core\Cache\CacheFactory
+    constructor:
+      namespace: "addons"
+      defaultLifetime: 600
```

Comment:
What's the default lifetime at the moment? I feel like 10 minutes wouldn't really achieve much since the jobs probably don't run that regularly anyway. We might need to go into the 1-3 day range to see any benefit from caching at all

Reply:
I'll set the lifetime to 6 hours. It may not be providing any benefit but data should be refreshed after 6 hours.